menubot/index.js

```diff
@@ -2,38 +2,12 @@ import { CharInfo } from "../bossroom";
 
 let rg = null;
 
-/**
- * Defines the type of character that the game should use for this bot.
- */
- export function getCharacterType() {
-  return CharInfo.type[1]; // fixed to rogue character
+export function configureBot(rg) {
+  rg.isSpawnable = false;
+  rg.characterType = CharInfo.type[1]; // fixed to rogue character
+  rg.lifecycle = "PERSISTENT";
```

Comment:
Oh @RG-nAmKcAz now that you are back, I wanted to chat about this at some point - I'm still having a hard time figuring out why the lifecycle flag is needed - can't users just assume they can do either? e.g. they can mark the bot as complete in the javascript code when they want, or they can shut it down from the Unity code, no flagging needed?

Reply:
PERSISTENT == you are responsible for stopping your own bot.  That means when `stopGame` is called on the unity side, your bot WILL NOT be stopped.  This is used for `startScenario` bots.

Reply:
menu bots are persistent

Reply:
Here's a scenario I can think of that we'd want persistence for:
* My bot clicks buttons to get my from the main menu into my game
* The bot doesn't do anything during the game itself
* Once the game ends, my bot clicks more buttons to close end-game screens and get back to the main menu
* Repeat

Reply:
Yes ! What @abeizer described is a good example.

Reply:
Got it, thank you! I have a few follow-up questions, but I don't like that I took over this PR to discuss XD so after my onboarding with Robb I'll send some messages in a slack channel.